src/frontend/src/storage/index.ts

```diff
@@ -269,12 +278,38 @@ export const getCancelledRpIds = async ({
 }: {
   userNumber: bigint;
   origin: string;
-}): Promise<Set<string | undefined>> => {
+}): Promise<{
+  cancelledRpIds: Set<string | undefined>;
+  lastShownAddCurrentDevicePage: number | undefined;
```

Comment:
This new field will be used to calculate whether the screen to add current device should be shown or not.